src/xtgeo/surface/regular_surface.py

```diff
@@ -2459,20 +2485,22 @@ def get_randomline(self, fencespec, hincrement=None, atleast=5, nextend=2):
         map resolution. If a Polygons() instance, this is automated if hincrement is
         None, and ignored if hincrement is False.
 
+        Returns a ndarray with shape (:, 2)
+
         Args:
-            fencespec (:obj:`~numpy.ndarray` or :class:`~xtgeo.xyz.polygons.Polygons`):
+            fencespec:
                 2D numpy with X, Y, Z, HLEN as rows or a xtgeo Polygons() object.
-            hincrement (float or bool): Resampling horizontally. This applies only
+            hincrement: Resampling horizontally. This applies only
                 if the fencespec is a Polygons() instance. If None (default),
                 the distance will be deduced automatically. If False, then it assumes
                 the Polygons can be used as-is.
-            atleast (int): Minimum number of horizontal samples (only if
+            atleast: Minimum number of horizontal samples (only if
                 fencespec is a Polygons instance and hincrement != False)
-            nextend (int): Extend with nextend * hincrement in both ends (only if
+            nextend: Extend with nextend * hincrement in both ends (only if
                 fencespec is a Polygons instance and hincrement != False)
+            sampling: Use "bilinear" (default) for interpolation or "nearest" for
+                snapping to nearest node.
 
-        Returns:
-            An array: ndarray2d (:, 2)
```

Comment:
Wouln't this look better in sphinx instead of a sentence above describing the return type?

Reply:
Will check (have started to use typing hints which can return the type as `def whatever(some: str) -> np.ndarray`

Reply:
Yes typing hints with automate